### ISSUE TITLE
Can fmdb update get the number of data effected

### ISSUE THREAD

**Issue by loumzy** [ccgus/fmdb#397]
if we can get the number of data when we use update；

**Comment by robertmryan** [comment]
If you want the number of rows that were affected by the last `UPDATE` statement, see `FMDatabase` method [`changes`](http://ccgus.github.io/fmdb/html/Classes/FMDatabase.html#//api/name/changes). If you're looking for something else, please clarify your question.

**Comment by loumzy** [comment]
Dear robertmryan:
Thank you for your reply.  My question is that When we use 'update' in database, the instruction will return the number of row it is effected when the new data to update is different from the old , and return '0' if the new data to update is same with the old. Dose 'fmdb' have this function?

**Comment by robertmryan** [comment]
No, but you could add a `WHERE` clause so it would only update records that needed updating:

    UPDATE x SET y = 'foo' WHERE y <> 'foo'

Then `changes` will return the number of records changed.

**Comment by loumzy** [comment]
Thank you very much
